.gitignore

```diff
@@ -42,6 +42,7 @@ log*.txt
 out/
 
 build/
+captures/
 .gradle/
 
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
My vim setup automatically adds a newline to the end of text files if they don't exist - I can remove if you want.